source/common/http/conn_manager_impl.h

```diff
@@ -615,6 +617,18 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
       // Used to track which filter is the latest filter that has received data.
       ActiveStreamEncoderFilter* latest_data_encoding_filter_{};
       ActiveStreamDecoderFilter* latest_data_decoding_filter_{};
+
+      bool decorated_propagate_ : 1;
+      // By default, we will assume there are no 100-Continue headers. If encode100ContinueHeaders
+      // is ever called, this is set to true so commonContinue resumes processing the 100-Continue.
+      bool has_continue_headers_ : 1;
+      bool is_head_request_ : 1;
```

Comment:
These need to be packed above with the other boolsn to get the wanted effect. Can you move them up there?